src/mscorlib/shared/System/TimeZoneInfo.Win32.cs

```diff
@@ -11,6 +11,20 @@
 using System.Threading;
 using Microsoft.Win32;
 
+#if CORERT
```

Comment:
I see... Either me or @tarekgh will hopefully work through the code to respect the guidelines https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md